docs/src/how-to/install/tls.md

```diff
@@ -40,6 +40,17 @@ The list of TLS ciphers for incoming SFT requests (and metrics) are defined in a
 SFTD deployed via kubernetes uses `kubernetes.io/ingress` for ingress traffic, configured in [ingress.yaml](https://github.com/wireapp/wire-server/blob/develop/charts/sftd/templates/ingress.yaml).
 Kubernetes based deployments make use of the settings from {ref}`ingress-traffic`.
 
+## Coturn (kubernetes)
+
+The list of TLS ciphers for "TLS over TCP" TURN are defined in the [coturn helm chart](https://github.com/wireapp/wire-server/blob/master/charts/coturn/)
+
+```{eval-rst}
+.. grepinclude:: ../charts/coturn/values.yaml ciphers:
+   :lines-before: 3
+   :lines-after: 0
+   :language: yaml
+```
```

Comment:
Doe we need the `eval-rst`?

The equivalent markdown should look something like

```

 ```{grepinclude} chiphers
---
lines-before: 3
lines-after: 0
language: yaml
---
...
```

Reply:
makes sense. Adjusted code to match that.